### PR TITLE
Make app compatible with Django 1.8 - 1.11 and remove Django 1.7 support.

### PR DESCRIPTION
**Pull request by somexpert**

### CODE REVIEW

**Comment by somexpert** [comment]
ping @ldko

**Comment by somexpert** [comment]
Bumping this to version 2.0.0 since we're removing Django 1.7 backwards compatibility.

File: tests/settings.py [hunk 1]
```diff
@@ -35,7 +35,6 @@
                 'django.template.context_processors.request',
                 'django.contrib.auth.context_processors.auth',
                 'django.contrib.messages.context_processors.messages',
-                'django.core.context_processors.request',
```
Comment (ldko) [inline tests/settings.py:38]: The README says to add this context_processor, should that be taken out of the README?

File: requirements/requirements-test.txt [hunk 2]
```diff
@@ -1,4 +1,4 @@
-tox>=2.1.1
-pytest>=2.8.2
-pytest-django>=2.9.0
-factory_boy>=2.6.0
+tox==3.3.0
+pytest==3.8.0
+pytest-django==3.4.2
+factory_boy==2.11.1
```
Comment (ldko) [inline requirements/requirements-test.txt:4]: I think we need to add flake8 to the requirements.

File: tests/settings.py (hunk 1 above)
Comment (ldko) [reply tests/settings.py:38]: I think it would be helpful to add `ALLOWED_HOSTS = ['0.0.0.0']` to tests/settings.py.

Comment (somexpert) [reply tests/settings.py:38]: "yes" and "will do!"

File: requirements/requirements-test.txt (hunk 2 above)
Comment (somexpert) [reply requirements/requirements-test.txt:4]: okie dokie

**Review by ldko — APPROVED** [review]

**Pull request merged**